src/CMetricsCalculator.cpp

```diff
@@ -34,6 +34,17 @@ is_eol_char(char c) {
 	return (c == '\r' || c == '\n');
 }
 
+char
+CMetricsCalculator::get_first_non_space_char(char terminated_c)
```

Comment:
`char_before` moves you back right?  So doesn't this function return the first **previous** non-space character?